### ISSUE TITLE
Spaces in Tag Value causes "Malformed input-No value separator" error

### ISSUE THREAD

**Issue by harlanbarnes** [bazaarvoice/cloudformation-ruby-dsl#34]
I realize this is such an edge case ... but I thought I'd bring it up in case anybody else runs across this. I'd fix it if I could, but it looks like it's bigger change that I'm comfortable with. I'm just happy to have this tool. :smile: 

We use some tags that have spaces in the value. For example:

```
tag Application: 'My App'
```

This might be a bad idea for other reasons, but at the moment, it's what we do and we've got a lot of existing tags in this format.

It looks like ```system``` calls escape spaces for the command line according to [this](http://stackoverflow.com/questions/23219932/ruby-system-arguments) and I'm guessing that's what's fouling up the ```cfn-create stack``` call. It might cause other problems with the ```diff``` and ```cfn-update-stack``` too even if the ```cfn-create-stack``` case could be worked around.

It didn't look like there's an obvious way to fix this without a lot of testing. So presuming that the spaces are just going to not be supported for now, I was wondering if you'd accept a PR that checks the tag values for spaces and causes an error that explains the problem and how to workaround (i.e. remove the spaces)? The reason I ask is the current error (using a tag example like the 'My App' above) isn't super obvious about the actual problem:

```
cfn-create-stack:  Malformed input-No value separator = found in App
```

Thanks. And again, great tool!

**Comment by temujin9** [comment]
I would gladly accept a PR that throws an error, for now; self-documenting code is a high priority. We'd obviously be happier if things like this actually worked as intended, so we'll probably leave this ticket open until we have a real solution, too.

**Comment by jonaf** [comment]
This should be resolved by #50
